README.md

```diff
@@ -251,7 +251,7 @@ several settings for `Rake::ExtensionTask`:
 ## Cross compilation - the future is now.
 
 Rake-compiler also provides a standardized way to generate, from either Linux
-or OSX, extensions and gem binaries for your Windows users!
+or OSX, extensions and gem binaries for your Windows and Linux users!
```

Comment:
I think that macOS users can't generate fat gems for Linux without rake-compiler-dock.

Reply:
Ah, Homebrew provides x86_64-elf-gcc.

How do users prepare Rubies for Linux on macOS?

Reply:
I think by specifying `rake-compiler cross-ruby HOST=x86_64-elf-gcc` or so. But I have little experience with MacOS only.

Reply:
`cross-ruby` expects only Mingw-w64 builds. For example, it uses `.exe` extension for Ruby binary.

rake-compiler doesn't support cross compiling fat gems for Linux. Could you revert related changes?

Reply:
> `cross-ruby` expects only Mingw-w64 builds.

Not by any means! It is pretty agnostic to the build target. Just the default is `mingw`, if no `HOST=triple` is given.

But you're right, `ruby.exe` is an [expected file target](https://github.com/rake-compiler/rake-compiler/blob/master/tasks/bin/cross-ruby.rake#L137). However this actually doesn't hurt - it's only that the file target is never satisfied so that the build is triggered in any case. Maybe we should fix this inaccuracy, now that `rake-compiler cross-ruby` is used for many different targets.

Reply:
It just works accidentally. It's not a "supported" feature. :-)

Could you revert this for now?